gapic/src/main/com/google/gapid/perfetto/views/CpuSlicesSelectionView.java

```diff
@@ -35,16 +37,18 @@
  * Displays information about a list of selected CPU slices.
  */
 public class CpuSlicesSelectionView extends Composite {
```

Comment:
Now that we have only Slices class, do we want to rename either this as Multi Selection view or the SliceSelectionView as SingleSliceSelectionView or something similar?

Reply:
Alternatively, we can have only the SlicesSelectionView composite and based on the count, show either the single selection or the multi-selection.

Reply:
Thanks for this great idea! I've done it following your second idea in "Address comments #2". File number is reduced thanks to your suggestion!